dell/xps/15-9500/default.nix

```diff
@@ -0,0 +1,8 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ./xps-common.nix
+  ];
```

Comment:
You can rename xps-common.nix to default.nix now and update the nvidia extension.